host/installer/installer.nsi

```diff
@@ -0,0 +1,196 @@
+/*
+Looking Glass - KVM FrameRelay (KVMFR) Client
+Copyright (C) 2017-2020 Geoffrey McRae <geoff@hostfission.com>
+https://looking-glass.hostfission.com
+
+This program is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License as published by the Free Software
+Foundation; either version 2 of the License, or (at your option) any later
+version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+Place, Suite 330, Boston, MA 02111-1307 USA
+*/
+
+
+;Include  
+!include "MUI2.nsh"
+!include "FileFunc.nsh"
+!include "LogicLib.nsh"
+!include "Sections.nsh"
+
+;Settings
+Name "Looking Glass Host"
+OutFile "..\build\looking-glass-host-setup.exe" 
+Unicode true
+RequestExecutionLevel admin
+ShowInstDetails "show"
+ShowUninstDetails "show"
+InstallDir "$PROGRAMFILES64\Looking Glass"
+
+!define MUI_ICON "..\..\resources\icon.ico"
+!define MUI_UNICON "..\..\resources\icon.ico"
+!define MUI_LICENSEPAGE_BUTTON "Agree"
+!define /file VERSION "..\..\VERSION"
+
+;Install and uninstall pages
+!insertmacro MUI_PAGE_LICENSE "..\..\LICENSE"
+!insertmacro MUI_PAGE_DIRECTORY
+!insertmacro MUI_PAGE_COMPONENTS
+!insertmacro MUI_PAGE_INSTFILES
+!insertmacro MUI_UNPAGE_CONFIRM
+!insertmacro MUI_UNPAGE_INSTFILES
+!insertmacro MUI_LANGUAGE "English"
+
+
+Function ShowHelpMessage
+	!define line1 "Command line options:$\r$\n$\r$\n"
+	!define line2 "/S - silent install (must be uppercase)$\r$\n"
+	!define line3 "/D=path\to\install\folder - Change install directory$\r$\n"
+	!define line4 "   (Must be uppercase, the last option given and no quotes)$\r$\n$\r$\n"
+	!define line5 "/startmenu - create start menu shortcut$\r$\n"
+	!define line6 "/desktop - create desktop shortcut$\r$\n"
+    !define line7 "/noservice - do not create a service to auto start and elevate the host"
```

Comment:
Formatting is inconsistent, tabs vs spaces. Please use the project standard, indents are two spaces for each level, no tabs.